include/astshim/Frame.h

```diff
@@ -1011,10 +1010,10 @@ class Frame : public Mapping {
     Thus input axis values corresponding to positions which are outside the
     Region will result in bad output axis values.
 
-    @param[in] first  The first frame in the compound frame (axes 1 - first.getNAxes())
+    @param[in]  next  The next frame in the compound frame (the final next.getNAxes() axes)
     @return a new CmpFrame
     */
-    CmpFrame over(Frame const &first) const;
+    CmpFrame under(Frame const &next) const;
```

Comment:
Both names are kind of confusing. Isn't this operation `X<1D> + Y<1D> -> XY<2D>`? Something like "join" might be more suggestive than "under".